tests/e2e/actor_reminder/actor_reminder_test.go

```diff
@@ -211,6 +231,90 @@ func TestActorReminder(t *testing.T) {
 			}
 		}
 
+		resp, err := utils.HTTPDelete(logsURL)
+		require.NoError(t, err)
+
+		t.Log("Done.")
+	})
+
+	t.Run("Actor reminder rename should succeed.", func(t *testing.T) {
+		var wg sync.WaitGroup
+		for iteration := 1; iteration <= numIterations; iteration++ {
+			wg.Add(1)
+			go func(iteration int) {
+				defer wg.Done()
+				t.Logf("Running iteration %d out of %d ...", iteration, numIterations)
+
+				for i := 0; i < numActorsPerThread; i++ {
+					actorID := fmt.Sprintf(actorIDRestartTemplate, i+(1000*iteration))
+					// Deleting pre-existing reminder
+					_, err = utils.HTTPDelete(fmt.Sprintf(actorInvokeURLFormat, externalURL, actorID, "reminders", reminderName))
```

Comment:
Please, use a different reminder name for this test. So, it does not conflict with the reminders created in the previous test. This can also help debug flaky tests later on.

Reply:
Done.